service/iot-service-samples/pnp-service-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/PnPNoComponentSample.java

```diff
@@ -0,0 +1,60 @@
+package samples.com.microsoft.azure.sdk.iot;
+
+import com.microsoft.azure.sdk.iot.service.devicetwin.*;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+public class PnPNoComponentSample {
+    private static final String propertyName = "targetTemperature";
+    private static final double propertyValue = 75.5;
+    private static final String methodName = "reboot";
+    private static final Long responseTimeout = TimeUnit.SECONDS.toSeconds(200);
+    private static final Long connectTimeout = TimeUnit.SECONDS.toSeconds(5);
+
+    public static void main(String[] args) throws Exception {
+        System.out.println("Starting sample...");
+        // get user inputs from environment variables
+        String iotHubConnectionString = System.getenv("IOTHUB_DEVICE_CONNECTION_STRING");
```

Comment:
Is this suppose to be DeviceConString or IotHub connection string?

Reply:
This is hub connection string. I will update the env variable more to be more appropriate.

Reply:
env var alignment with other SDKs:

Device | IOTHUB_DEVICE_CONNECTION_STRING
-- | --
Service | IOTHUB_CONNECTION_STRING

@bikamani FYI, as well